pkg/api/component.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/tokamak-network/tokamak-titond-backend/pkg/model"
+)
+
+func (t *TitondAPI) CreateComponent(data *model.Component) (*model.Component, error) {
+	fmt.Println("Create component api handler")
+	return data, nil
+}
+
+func (t *TitondAPI) GetComponentByType(networkID uint, componentType string) (*model.Component, error) {
```

Comment:
I think it is better to move this function to database! It is just query of database. 🙇🏼‍♂️